Packaging/vita/sce_sys/livearea/contents/template.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<livearea style="psmobile" format-ver="01.00" content-rev="1">
+ 
+	<livearea-background>
+    	<image>bg.png</image>
+	</livearea-background>
+
+	<gate>
+		<startup-image>startup.png</startup-image>
+	</gate>
+
+	<frame id="frame1">
+		<liveitem>
+			<image>logo0.png</image>

```

Comment:
Is it correct that the propper size here would be 480x272? How does the menu handle images of smaller sizes?

Reply:
Nah, frames can be any size.

Reply:
I mean, two frames split livearea in two parts, but frame content can contain images of any size. They are drawn as-is with their original size. (so image smaller than frame won't take all frame area)

Reply:
Thanks :)